presentation/src/main/java/com/exallium/mvvmcleanexample/presentation/utils/notifiable/BaseNotifiableObservable.kt

```diff
@@ -0,0 +1,67 @@
+package com.exallium.mvvmcleanexample.presentation.utils.notifiable
+
+import android.databinding.Bindable
+import android.databinding.Observable
+import android.databinding.PropertyChangeRegistry
+
+class BaseNotifiableObservable : NotifiableObservable {
```

Comment:
I see.  Hmm.  I think if it's preventing repeated code, it's probably ok to keep around.  I have an adversion to boilerplate.